docs/data/material/components/badges/badges.md

```diff
@@ -74,10 +74,11 @@ You should provide a full description, for instance, with `aria-label`:
 
 ## Experimental APIs
 
-### Material You version
+### Material 3 version
 
 The default Material UI Badge component follows the Material Design 2 specs.
-To get the Material You ([Material Design 3](https://m3.material.io/)) version, use the new experimental `@mui/material-next` package:
+To get the [Material 3](https://m3.material.io/) version, use the new experimental `@mui/material-next` package.
+Visit the [Material 3 Components](/material-ui/guides/material-3-components/) guide for instructions on how to do so.
```

Comment:
I guess we could simplify these lines here, somehow — there are too many mentions of "Material" 😅 If I were to throw in a suggestion, it'd be something along these lines, maybe:

> The default Material UI {name} component follows the Material Design 2 specs.
> To get the Material 3 version, use the new experimental `@mui/material-next` package.

// installation code block here //

> For more instructions on how to use it, visit the detailed guide.

Something like this.

Reply:
Agree 👍🏼 implemented